### PR TITLE
Ask for confirmation when removing a table.

### LINKED ISSUE #152: Table removal should ask for confirmation.
It is very easy to accidentally click the "Remove" item in the menu.

### PR DESCRIPTION
**Pull request by cristianberneanu**
Closes #152.

### CODE REVIEW

File: src/AdminTab/TableList.tsx [hunk 1]
```diff
@@ -41,9 +41,17 @@ const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMet
       >
         {t('New SQL query')}
       </Menu.Item>
-      <Menu.Item icon={<DeleteOutlined />} key={`${table.name}-remove`} onClick={() => removeTable(table.name)}>
-        {t('Remove')}
-      </Menu.Item>
+      <Popconfirm
```
Comment (pdobacz) [inline src/AdminTab/TableList.tsx:44]: I just remembered (late :( ) I think Edon had the idea to implement an ohnosecond "Undo" button, showing up in the notification popup, somewhat like what you have in Gmail.

Another (late as well) thought is to maybe separate the ":wastebasket: Remove" separated by a separator in the menu, so it's harder to misclick?

File: assets/locales/en/translation.json [hunk 2]
```diff
@@ -61,6 +61,9 @@
       "TableDropdown": {
         "New SQL query": "New SQL query",
         "Remove": "Remove",
+        "Are you sure to delete table `{{name}}`?": "Are you sure to delete table `{{name}}`?",
```
Comment (pdobacz) [inline assets/locales/en/translation.json:64]: I'd make the texts shorter like so:
"Remove table {{name}}?"
"Remove"
"Cancel"

I think yes/no modals are to be avoided. We're using "remove" in the GUI, and "delete" in my ear sounds more like destruction of the data, not just removing from some form of "registry" which our app kinda is.

**Review by pdobacz — DISMISSED** [review]
some notes on wording mainly

File: src/AdminTab/TableList.tsx (hunk 1 above)
Comment (cristianberneanu) [reply src/AdminTab/TableList.tsx:44]: Hm, I'm not sure what to do here... Do you want me to add a separator now? There are only two items in the menu, not sure if it makes sense...

Comment (pdobacz) [reply src/AdminTab/TableList.tsx:44]: Sorry about being vague here. I'm just tossing alternative ideas, but I don't have a strong opinion. Also the "Undo" thing is much more work. So if neither appeals to you now, let's just leave them for later and merge as is.

**Review by pdobacz — APPROVED** [review]

**Pull request merged**